packages/analytics/__tests__/analytics.test.ts

```diff
@@ -843,10 +877,18 @@ describe('Analytics', function () {
       expect(initiateOnDeviceConversionMeasurementWithEmailAddress).toBeDefined();
     });
 
+    it('`initiateOnDeviceConversionMeasurementWithHashedEmailAddress` function is properly exposed to end user', function () {
```

Comment:
Do you mind just adding the same unit test you've done above for modular API as well? 

e.g.
```js
 expect(() => initiateOnDeviceConversionMeasurementWithHashedEmailAddress(analytics, '+1234567890');
      ).toThrowError(
        "firebase.analytics().initiateOnDeviceConversionMeasurementWithHashedPhoneNumber(*) 'hashedPhoneNumber' expected a sha256-hashed value of a phone number in E.164 format.",
      );
    });

```

Reply:
Done and rebased! [830cec4](https://github.com/invertase/react-native-firebase/pull/7963/commits/830cec42e9f77643e80f1ade4efffd4335c39663)